modules/cli/pages/cli-intro.adoc

```diff
@@ -39,46 +39,117 @@ You can collect diagnostics through the command-line interface by using the xref
-* macOS: https://packages.couchbase.com/releases/{tools-ver}/couchbase-server-tools_{tools-ver}-macos_x86_64.zip[]
-* macOS arm64: https://packages.couchbase.com/releases/{tools-ver}/couchbase-server-tools_{tools-ver}-macos_arm64.zip[]
-* Windows: https://packages.couchbase.com/releases/{tools-ver}/couchbase-server-tools_{tools-ver}-windows_amd64.zip[]
+* Linux: https://packages.couchbase.com/releases/{tools-ver}/couchbase-server-admin-tools_{tools-ver}-linux_x86_64.tar.gz[]
+* Linux aarch64: https://packages.couchbase.com/releases/{tools-ver}/couchbase-server-admin-tools_{tools-ver}-linux_aarch64.tar.gz[]
+* macOS: https://packages.couchbase.com/releases/{tools-ver}/couchbase-server-admin-tools_{tools-ver}-macos_x86_64.zip[]
+* macOS arm64: https://packages.couchbase.com/releases/{tools-ver}/couchbase-server-admin-tools_{tools-ver}-macos_arm64.zip[]
+* Windows: https://packages.couchbase.com/releases/{tools-ver}/couchbase-server-admin-tools_{tools-ver}-windows_amd64.zip[]
 
 Unzip or untar the packages, and the binaries are ready to run.
 For example:
 
 [source,console,subs="attributes+"]
 ----
-wget https://packages.couchbase.com/releases/{tools-ver}/couchbase-server-tools_{tools-ver}-linux_x86_64.tar.gz
+wget https://packages.couchbase.com/releases/{tools-ver}/couchbase-server-admin-tools_{tools-ver}-linux_x86_64.tar.gz
 
-tar -xf couchbase-server-tools_{tools-ver}-linux_x86_64.tar.gz
+tar -xf couchbase-server-admin-tools_{tools-ver}-linux_x86_64.tar.gz
 ----
 
 Each package also contains a `README` file and a copy of the software license.
 
 NOTE: On Windows, you must have a recent version of the Microsoft Visual {cpp} Redistributable runtime libraries installed.
 If you do not have these libraries installed, download them from https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170[Microsoft Visual {cpp} Redistributable latest supported downloads].
 
+
 [#version-compatibility]
 == Version Compatibility
 
 The versions of the utilities installed by the tools package are the same as the corresponding Couchbase Server installation package.
 
-The {tools-ver} `cbimport`, `cbexport`, `cbbackupmgr`, and `cbq` utilities are compatible with the following Couchbase Server versions:
+=== For Server Developer Tools Packages
+
+The {tools-ver} version of `cbimport`, `cbexport`, `cbbackupmgr`, `cbq`, `couchbase-cli`, `mctimings`, `cbstats`, and `mcstat` utilities are compatible with the following Couchbase Server versions:
+
+* 7.6.0, 7.6.1, 7.6.2, 7.6.3
+* 7.2.x
+* 7.1.x
+* 7.0.x
+
+The 7.6.2 version of `cbimport`, `cbexport`, `cbbackupmgr`, and `cbq` utilities are compatible with the following Couchbase Server versions:
 
 * 7.6.0, 7.6.1
 * 7.2.x
 * 7.1.x
 * 7.0.x
+
+So, you can download and use the latest version of these utilities (`cbimport`, `cbexport`, `cbbackupmgr`, `cbq`, `couchbase-cli`, `mctimings`, `cbstats`, and `mcstat`) with the previous Couchbase Server versions.
```

Comment:
So, you can download and use the latest version of these utilities (`cbimport`, `cbexport`, `cbbackupmgr`, `cbq`, `couchbase-cli`, `mctimings`, `cbstats`, and `mcstat`) with the previous Couchbase Server versions.

Should be:
So, you can download and use the latest version of these utilities (`cbimport`, `cbexport`, `cbbackupmgr`, and `cbq`) with the previous Couchbase Server versions.